README.md

```diff
@@ -74,17 +85,17 @@ You bet. All of our custom builders accept "command line style" arguments with t
 
 ##### How is the build process logged?
 
-Each of our custom builders produces a log of its process in the same directory as the first of its targets. Each log is named `sconscript.log` by default, and you can insert custom text between `sconscript` and `.log` by passing it as a string through the builder's `log_ext` keyword argument. It's similar to the way that you specify sources and targets, except that the `log_ext` argument must be a string. It can be important to specify the `log_ext` argument for builders that produce logs in the same directory. The default naming convention allows builders to overwrite one another's logs, so you'll only have a log of the last process to finish. 
+Each of our custom builders produces a log of its process in the same directory as the first of its targets. Each log is named `sconscript.log` by default, and you can insert custom text between `sconscript` and `.log` by passing it as a string through the builder's `log_ext` keyword argument. It's similar to the way that you specify sources and targets, except that the `log_ext` argument must be a string. It can be important to specify the `log_ext` argument for builders that produce logs in the same directory. The default naming convention leads builders to overwrite one another's logs, so you'll only have a log of the last process to finish. 
 
 After all the steps in the build are completed, we'll comb through the directory and look for for any files named `sconscript*.log`. These logs will be concatenated—with the earliest completed ones first and all logs with errors on top. We'll store this concatenated log at the root of the repository in `sconstruct.log`. 
 
 ##### Can I write my paper in LaTeX instead of LyX?
 
 We don't have a custom builder for LaTeX. You can still write in it, but you will have to use [SCons's native builder](http://www.scons.org/doc/0.96.91/HTML/scons-user/a5334.html). You can still use our custom table builder to fill LaTeX tables. 
 
-##### Can I release to GitHub?
+##### Can I release my repository?
 
-Yes, see [here](https://github.com/gslab-econ/gslab_python/tree/master/gslab_scons) for directions on making a release.
+Yes, our [custom tool](https://github.com/gslab-econ/gslab_python/tree/master/gslab_scons) allows you to release a repository to GitHub and a local destination specified in `user-config.yaml`. Local releases can be transfered to a remote manually (e.g., using rsync or rclone) or automatically by specifying a local destination that's synced to a remote (e.g., standard Dropbox behavior). Every file intended for release should be added to the `release` directory. Files intended for local release only should be added to `.gitignore`. Our tool will transfer everything in `release` to the local destination and create a [GitHub release](https://help.github.com/articles/creating-releases/) with all the versioned files—those not added to `.gitignore`—in `release`. 
```

Comment:
I'd put `rsync` and `rclone` in code script. 

"Every file intended for release should be added to the `release` directory. Files intended for local release only should be added to `.gitignore`."

This sentence is technically incorrect. Releasing to "DropBox" shouldn't be thought of as "local release." I vote we change the terminology to "drive release".

I'd also split this paragraph into two starting in that sentence.